src/components/Shared/SpectralSettings/SpectralSettingsComponent.tsx

```diff
@@ -14,6 +14,7 @@ export class SpectralSettingsComponent extends React.Component<{
     disable: boolean;
     disableChannelOption?: boolean;
     secondaryAxisCursorInfoVisible?: boolean;
+    label?: string;
```

Comment:
I suggest we rename it to `customLabel`.

Reply:
fixed